javascript-typing-tutor/styles.css

```diff
@@ -0,0 +1,61 @@
+h1 {
+  font-size: 3rem;
+}
+
+button {
+  border-radius: 5px;
+  background-color: rgb(33, 129, 189);
+  color: rgb(220, 229, 233);
+  border-style: none;
+  font-size: 1.2rem;
+  font-weight: bolder;
+  height: 50px;
+  width: 150px;
+  cursor: pointer;
+}
+
+.row {
+  display: flex;
+}
+
+.text-center {
+  justify-content: center;
+}
+
+.margin-top {
+  margin-top: 200px;
+}
+
+.off {
+  color: black;
+}
+
+.next {
+  color: black;
+  border-bottom: solid 2px black;
+}
+
+.green {
+  color: rgb(34, 212, 34);
+}
+
+.red {
+  color: red;
+  border-bottom: solid 2px red;
+}
+
+.dialog-off {
+  display: none;
+}
+
+.dialog-on {
+  display: flex;
+}
+
+.score-off {
+  display: none;
+}
+
+.score-on {
+  display: flex;
+}
```

Comment:
These rules are copies of each other, you should make one rule for each piece of styling and reuse it. In general, it is a bad idea to have two rules with the same styling being applied to them.